app/src/org/commcare/android/database/global/models/ConnectKeyRecord.java

```diff
@@ -7,18 +7,14 @@
 @Table(ConnectKeyRecord.STORAGE_KEY)
 public class ConnectKeyRecord extends Persisted {
     public static final String STORAGE_KEY = "connect_key";
-
-    @Persisting(1)
-    String userId;
```

Comment:
curious why we removed `userID` ? Is it because only one user can exist on a device ?

Reply:
That's correct. If we decide later to support multiple CID users on a device, this may or may not need to change depending on whether we decide to create separate ConnectID databases for each user or use on DB for all.